datajoint/hub.py

```diff
@@ -0,0 +1,43 @@
+import re
+import requests
+from requests.adapters import HTTPAdapter
+from datajoint.errors import DataJointError
+
+HUB_PROTOCOL = 'hub://'
+REQUEST_PROTOCOL = 'https://'
+API_ROUTE = '/api'
+API_TARGETS = dict(GET_DB_FQDN='/v0.000/get_database_fqdn')
+
+session = requests.Session()
+session.mount('http://', HTTPAdapter(max_retries=3))
+
+
+def get_host(host):
+    hub_path = re.findall('/[:._\-a-zA-Z0-9]+', host)
+    if re.match(HUB_PROTOCOL, host) and len(hub_path) > 2:
+        try:
+            resp = session.post('{}{}{}{}'.format(
+                REQUEST_PROTOCOL, hub_path[0][1:], API_ROUTE, API_TARGETS['GET_DB_FQDN']),
+                data={'org': hub_path[1][1:], 'project': hub_path[2][1:]}, timeout=10)
```

Comment:
Can we change these to org_name and project_name?
Thinking that's cleaner w/r/t attribute naming on the backend.

Reply:
Makes sense. I will include this into the next commit.